CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # @digitalbazaar/data-integrity Changelog
 
+## 2.1.0 - 2024-2-13
```

Comment:
At a minimum, the day here needs to be left as `dd` to allow the release manager to set it during the release process. Our release process requires a commit to this file so it needs something to be changed during that process -- and we use the release date for that (because it's a natural thing to do).

```suggestion
## 2.1.0 - 2024-02-dd
```